schemas/com.snowplowanalytics.sauna.commands/envelope/jsonschema/1-0-0

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a Sauna command's envelope",
+	"self": {
+		"vendor": "com.snowplowanalytics",
+		"name": "envelope",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"commandId": {
+			"type": "string",
+			"pattern": "^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$"
```

Comment:
This regexp matches [UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_.28random.29) identifiers:

`xxxxxxxx-xxxx-4xxx-%xxx-xxxxxxxxxxxx`

- `x`: any hexadecimal digit (`0`-`9`, `A`-`F`).
- `%`: one of `8`, `9`, `a` or `b`.